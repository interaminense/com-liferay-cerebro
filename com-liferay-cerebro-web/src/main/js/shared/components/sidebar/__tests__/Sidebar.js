import Sidebar from '../index';

describe('Sidebar', () => {
	let component;

	afterEach(() => {
		if (component) {
			component.dispose();
		}
	});

	it('should render', () => {
		component = new Sidebar();

		expect(component).toMatchSnapshot();
	});
});