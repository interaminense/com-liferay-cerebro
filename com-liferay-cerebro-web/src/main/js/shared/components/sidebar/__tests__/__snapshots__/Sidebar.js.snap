// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar should render 1`] = `
<div class="sidebar-root">
  <div class="header">
    <svg class="icon-root lexicon-icon lexicon-icon-dxp-logo-iso icon-size-md">
      <use xlink:href="/images/lexicon/icons.svg#dxp-logo-iso"></use>
    </svg>
    <svg class="icon-root lexicon-icon lexicon-icon-dxp-logo sidebar-hide">
      <use xlink:href="/images/lexicon/icons.svg#dxp-logo"></use>
    </svg>
  </div>
  <div class="sidebar-nav">
    <ul class="nav">
      <li class="sidebar-item">
        <a href="/home">
          <svg class="icon-root lexicon-icon lexicon-icon-dxp-home">
            <use xlink:href="/images/lexicon/icons.svg#dxp-home"></use>
          </svg>
          <span class="sidebar-hide">Home</span>
        </a>
      </li>
      <li class="sidebar-item">
        <a href="/contacts">
          <svg class="icon-root lexicon-icon lexicon-icon-dxp-contacts">
            <use xlink:href="/images/lexicon/icons.svg#dxp-contacts"></use>
          </svg>
          <span class="sidebar-hide">Contacts</span>
        </a>
      </li>
      <li class="sidebar-item">
        <a href="/assets">
          <svg class="icon-root lexicon-icon lexicon-icon-dxp-assets">
            <use xlink:href="/images/lexicon/icons.svg#dxp-assets"></use>
          </svg>
          <span class="sidebar-hide">Assets</span>
        </a>
      </li>
      <li class="sidebar-item">
        <a href="/touchpoints">
          <svg class="icon-root lexicon-icon lexicon-icon-dxp-touchpoints">
            <use xlink:href="/images/lexicon/icons.svg#dxp-touchpoints"></use>
          </svg>
          <span class="sidebar-hide">Touchpoints</span>
        </a>
      </li>
      <li class="sidebar-item">
        <a href="/campaigns">
          <svg class="icon-root lexicon-icon lexicon-icon-dxp-campaigns">
            <use xlink:href="/images/lexicon/icons.svg#dxp-campaigns"></use>
          </svg>
          <span class="sidebar-hide">Campaigns</span>
        </a>
      </li>
    </ul>
  </div>
  <div class="footer">
    <ul class="nav">
      <li class="sidebar-item">
        <a href="/settings">
          <svg class="icon-root lexicon-icon lexicon-icon-cog">
            <use xlink:href="/images/lexicon/icons.svg#cog"></use>
          </svg>
          <span class="sidebar-hide">Settings</span>
        </a>
      </li>
    </ul>
  </div>
</div>
`;
